lib/process-services/src/lib/process-list/components/start-process.component.ts

```diff
@@ -127,6 +127,7 @@ export class StartProcessInstanceComponent implements OnChanges, OnInit, OnDestr
 
     isProcessDefinitionsLoading = true;
     isAppsLoading = true;
+    movedNodeToPS;
```

Comment:
Can we add type here
```suggestion
    movedNodeToPS: FormValues;
```